### PR TITLE
Fix my-account logout 1.x

### PR DESCRIPTION
**Pull request by bivillar**
#### What did you change? \*
Added the Authstate Tag.

#### Why? \*
The logout needs this tag to work.

#### How to test it? \*
https://logout--boticario.myvtex.com/
log in and then try to logout from my-account.

#### Types of changes \*

- [x] Bug fix (a non-breaking change which fixes an issue)
- [ ] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical improvements

### CODE REVIEW

File: react/components/Menu/index.tsx [hunk 1]
```diff
@@ -4,7 +4,7 @@ import React, { Component, Fragment } from 'react'
 import { injectIntl, FormattedMessage, InjectedIntlProps } from 'react-intl'
```
Comment (salesfelipe) [inline react/components/Menu/index.tsx:4]: Changelog?

Comment (bivillar) [reply react/components/Menu/index.tsx:4]: It's there

File: react/components/Menu/index.tsx [hunk 2]
```diff
@@ -44,40 +44,42 @@ class Menu extends Component<Props, { isModalOpen: boolean }> {
...
+                      label: intl.formatMessage({
+                        id: 'vtex.store-messages@0.x::logoutModal.cancel',
+                      }),
+                    }}
+                    isOpen={this.state.isModalOpen}
+                    onClose={this.handleModalToggle}>
+                    <span className="t-heading-5 pa6">
+                      <FormattedMessage id="vtex.store-messages@0.x::logoutModal.title" />
+                    </span>
+                  </ModalDialog>
```
Comment (salesfelipe) [inline react/components/Menu/index.tsx:78]: I think this can be outside the AuthState

Comment (bivillar) [reply react/components/Menu/index.tsx:78]: I think it looks more organized like that. Why should I move it out?

Comment (salesfelipe) [reply react/components/Menu/index.tsx:78]: Less rerenders on this node...

Comment (bivillar) [reply react/components/Menu/index.tsx:78]: It does need to be inside 😞. It uses the logout prop from AuthService (line 62).

**Pull request merged**